fix/fixer_azure_exclude_from_latest.go

```diff
@@ -0,0 +1,58 @@
+package fix
+
+import (
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// FixerAzureExcludeFromLatest fix the spelling of "exclude_from_latest"
+// template in an Azure builder
+type FixerAzureExcludeFromLatest struct{}
```

Comment:
The fixer needs to be registered with packer in fixer.go. So that Packer knows about the fixer. 
You can achieve that by appending the new fixer to the data structures  in this file https://github.com/hashicorp/packer/blob/157b5cdc2845e617fd02463471b5a78da0d84b58/fix/fixer.go